common/script_values/00_men_at_arms_values.txt

```diff
@@ -365,3 +315,177 @@ culture_ai_weight_heavy_cavalry = {
 }
 
 # Warcraft
+
+culture_ai_weight_caster = {
+	value = 0
+	
+	if = { # Cultures that favours casters
+		limit = {
+			OR = {
+				culture_group = culture_group:human_group
+				culture_group = culture_group:dwarf_group
```

Comment:
Why dwarves?
Why not elves?
I think only elves and Dalaran should prefer casters.